tests/phpunit/unit-tests/admin/class-llms-test-admin-page-status.php

```diff
@@ -0,0 +1,241 @@
+<?php
+/**
+ * Test Admin Status page
+ *
+ * @package LifterLMS/Tests/Admin
+ *
+ * @group admin
+ * @group status
+ *
+ * @since [version]
+ */
```

Comment:
not totally sure because we don't use a cs for the tests, but I guess we also use to add the `@version` tag, right?

Reply:
@eri-trabiccolo I've stopped adding `@version` to test files since it's redundant (eg: the test files dont' have a fileheader and a class definition). I should probably document it but I really don't care.